x/bank/src/keeper/mod.rs

```diff
@@ -287,16 +92,20 @@ impl<SK: StoreKey, PSK: ParamsSubspaceKey, AK: AuthKeeper<SK, M>, M: Module>
     pub fn init_genesis<DB: Database>(
         &self,
         ctx: &mut InitContext<'_, DB, SK>,
-        genesis: GenesisState,
+        mut balances: Vec<Balance>,
+        params: BankParams,
+        denom_metadata: Vec<Metadata>,
     ) {
-        // TODO:
-        // 1. cosmos SDK sorts the balances first
-        // 2. Need to confirm that the SDK does not validate list of coins in each balance (validates order, denom etc.)
-        // 3. Need to set denom metadata
-        self.bank_params_keeper.set(ctx, genesis.params);
+        // 1. cosmos SDK sorts the balances first - Make sure that rust ordering gives same result
```

Comment:
This should still be a TODO

Reply:
Okay. But I checked and only 1 is worth todo, others correct

Reply:
Added TODO